src/base/bittorrent/session.cpp

```diff
@@ -774,12 +775,12 @@ bool Session::editCategory(const QString &name, const QString &savePath)
     m_categories[name] = savePath;
     m_storedCategories = map_cast(m_categories);
     if (isDisableAutoTMMWhenCategorySavePathChanged()) {
-        foreach (TorrentHandle *const torrent, torrents())
+        for (TorrentHandle *const torrent : qAsConst(m_torrents))
```

Comment:
you should not replace the use of `torrents()` by `m_torrents`, those are different objects, or you should make another commit for it.

Reply:
Oops this was a copy paste mistake i'll revert these.